websockets/websocket-next-oidc/src/main/java/io/quarkus/ts/websocketNext/endpoints/AuthenticatedWebSocket.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.ts.websocketNext.endpoints;
+
+import jakarta.inject.Inject;
+
+import io.quarkus.oidc.BearerTokenAuthentication;
+import io.quarkus.security.identity.SecurityIdentity;
+import io.quarkus.websockets.next.OnTextMessage;
+import io.quarkus.websockets.next.WebSocket;
+
+@WebSocket(path = "/bearer")
+@BearerTokenAuthentication
```

Comment:
Just for record, this was implemented in 3.21 https://github.com/quarkusio/quarkus/pull/46161 so let's make sure this portion is not backported.

Reply:
basically it does 2 things: enforces authenticated access and reduced available mechanisms to the OIDC bearer auth so I am not sure from your tests below that they fail without this